_gsocprojects/2020/project_SWAN.md

```diff
@@ -0,0 +1,11 @@
+---
+project: SWAN
+layout: default
+logo: swan_logo.svg
```

Comment:
We have a different logo naming convention now: SWAN-logo.svg. Please copy images/swan_logo.svg  to  images/SWAN-logo.svg and use the new name

Reply:
Pls do not delete the old logo file, it is in use. If you can provide .png instead of .svg even better

Reply:
@ebocchi you may need to rebase on master and fix the conflicts on mentors.md before re-pushing

Reply:
It should be fixed now. Thanks!